dags/openshift_nightlies/docs/secrets.md

```diff
@@ -0,0 +1,220 @@
+# Secret Variables
+
+While storing the Task configurations in Git is powerful, it's biggest gap is that it's a terrible place to store configurations you wish to keep secret. This could include things such as:
+
+* Cloud Account Credentials
+* Username/Passwords
+* SSH Keys
+
+To resolve this, Airflow lets you [define variables](https://airflow.apache.org/docs/apache-airflow/stable/howto/variable.html) in the UI/CLI and pulled through the Airflow SDK. 
+
+
+
+# Sailplane Vault
+
+For users leveraging the `playground` or `tenant` mode of installation benefit from the fact that Airflow instances created with those modes are auto-wired to connect to our [Vault](https://www.vaultproject.io/) instance. This ensures Airflow will work OOTB as our Vault has all of the required variables defined in it. 
+
+## Overriding Vault Variables
+
+If Airflow is connected to a Vault instance, those variables will take precendence over any defined locally within Airflow. In order to allow users to override the variables with their own configurations, users can create a singular `overrides` variable in Airflow that is a JSON blob with mappings of the variable key to the new desired value. 
+
+Example `overrides` variable:
+
+```json
+
+{
+    "ansible_orchestrator": {
+        "orchestration_host": "my_custom_host",
+        "orchestration_user": "my_custom_user"
+    },
+    "elasticsearch": "my_custom_elastic_url"
+}
+
+```
+
+
+
+
+
+# Supported Variables
```

Comment:
Its a good idea to list vars here, but keeping it up-to-date will be a challenge :P Let's keep an eye on this list.

Reply:
yeah I plan to block PRs if they add a new variable w/o updating the doc :D